_posts/en/newsletters/2019-09-25-newsletter.md

```diff
@@ -78,7 +78,24 @@ answers made since our last update.*
 endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME
+- [Why does this coinbase transaction have two op_return outputs?]({{bse}}90127)
+Murch outlines that while there are some restrictions on coinbase transactions,
+there are no special script or quantity restrictions regarding coinbase
+transaction outputs.
+
+- [What happens to transactions included in invalid blocks?]({{bse}}90026)
+RedGrittyBrick and Murch explain that a transaction's inclusion in an invalid block
+does not have any impact on the validty of that transaction or its ability to
+be confirmed in subsequent blocks.
+
+- [Why was 550 (MiB) chosen as a minimum storage size for prune mode?]({{bse}}90140)
+Behrad Khodayar answers his own question about why 550 (MiB) was chosen
+as a minimum storage size for pruned nodes. The storage amount was chosen
+to keep 288 blocks, about two days worth, on disk.
```

Comment:
Maybe say "...keep 288 blocks worth of state transition data, about two days worth, on disk" as the data included in this calculation is both blocks and undo data (undo data is the UTXOs each block spends and which are then removed from the UTXO set; if the block is later forked off the chain, those UTXO entries need to be re-added to the database).

The original answer doesn't note this and I'm not sure we do either---since the question is about why the original number was chosen---but the two day value no longer stands as the maximum size of blocks was increased by segwit (also the typical size of undo data, although I think the adversarial size remains the same).

Reply:
Took your suggestion and also added "originally" and "pre-segwit". You’re welcome to push a change if there is a better wording.